apiv2/test/client/provider_test.go

```diff
@@ -218,3 +218,50 @@ func TestProviderList_ListEmpty(t *testing.T) {
 	assert.Equal(t, http.StatusOK, resList.StatusCode())
 	assert.Equal(t, 1, len(resList.JSON200.Providers))
 }
+
+func TestProvider_Patch(t *testing.T) {
```

Comment:
I am confused - why do we have this test? We blocked on purpose UPDATE for providers and we do other resources too

Reply:
Still on final review of all methods/fields.

Reply:
Finished.